src/mocks/data.ts

```diff
@@ -425,6 +425,85 @@ export const searchResults: SearchResults = {
   ],
 };
 
+/**
+ * ARS API
+ */
+const startDate = new Date();
+const endDate = new Date(startDate);
+endDate.setFullYear(endDate.getFullYear() + 1);
+const access_starts = startDate.toISOString();
+const access_ends = endDate.toISOString();
+
+export const accessRequests = [
+  {
+    id: '62bcc452-a70b-47c1-9870-55da40d8e45f',
+    user_id: 'j.doe@ghga.de',
```

Comment:
```suggestion
    user_id: 'doe@test.dev',
```

Reply:
The emails have been changed in the other mock data, we should adapt here as well.

We should also add one or more access requests for Jane Roe (roe@test.dev) that can be later shown in the access request manager.